index.html

```diff
@@ -40,42 +45,26 @@
       </header>
 
       <section class="heading-section display-flex">
-        <h1 id="main-heading">
-          Hey there. I'm Ulises <br /><strong>I'm a Software Developer</strong>
-        </h1>
-
-        <p>
-          I can help you build a product, feature or website Look through some
-          of my work and experience! If you like what you see and have a project
-          you need coded, don't hestiate to contact me.
-        </p>
+        <h1 id="main-heading">Hey there. I'm Ulises<strong>I'm a Software Developer</strong></h1>
+        
+        <p>I can help you build a product, feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don't hestiate to contact me.</p>
 
         <ul id="icons-div" class="display-flex">
-          <li>
-            <a><img src="images/twitter.png" alt="twitter icon" /></a>
-          </li>
-          <li>
-            <a><img src="images/linkedin-icon.png" alt="linkedin icon" /></a>
-          </li>
-          <li>
-            <a><img src="images/medium.png" alt="A big M icon" /></a>
-          </li>
-          <li>
-            <a><img src="images/github.png" alt="github icon" /></a>
-          </li>
-          <li>
-            <a><img src="images/angellist.png" alt="two tumbs up icon" /></a>
-          </li>
+          <li><a><img src="images/twitter.png" alt="twitter icon" /></a></li>
+          <li><a><img src="images/linkedin-icon.png" alt="linkedin icon" /></a></li>
+          <li><a><img src="images/medium.png" alt="A big M icon" /></a></li>
+          <li><a><img src="images/github.png" alt="github icon" /></a></li>
+          <li><a><img src="images/angellist.png" alt="two tumbs up icon" /></a></li>
         </ul>
       </section>
     </div>
 
     <section class="works-section" id="portafolio">
```

Comment:
- [ ] The projects section should be created dynamically with the information from the projects array (all the HTML of the section is created when the page loads). This requires changes in existing code - already created HTML code for the projects list should be removed, and instead it should be added via JavaScript

Reply:
@Ismailco Hello!! Thank you so much for the review, would you mind clarifying more about this part? I'm not sure if you mean to remove the images, or what exact part has to be created dynamically, I'd appreciate it a lot!

Reply:
@ulisesedu7, as a requirement you need to remove the whole project section from your HTML file and instead Create it dynamically using JavaScript

Reply:
@Ismailco Thanks for clarifying, just to know if I'm getting it right, you mean the project section pop-up window, right? Does that have to be removed from the HTML to be created with JS instead?

Reply:
Gotcha Thanks @Ismailco!